src/main/java/com/codejoust/main/model/report/GameReport.java

```diff
@@ -0,0 +1,55 @@
+package com.codejoust.main.model.report;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+import com.codejoust.main.model.User;
+import com.codejoust.main.model.problem.Problem;
+
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+
+import lombok.AccessLevel;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Entity
+@Getter
+@Setter
+public class GameReport {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+
+    private String gameReportId;
```

Comment:
Should this be auto generated? Wondering if we're able to use a UUID here similar to in other entities

Reply:
Yes it should - I made this change in the next branch `GenerateReport`, but updated it here as well. 👍

Reply:
Yes this should! I actually did that in the [next branch](https://github.com/CodeJoustHQ/main/blob/GenerateReport/src/main/java/com/codejoust/main/model/report/GameReport.java), but I'll update it here as well. 👍